### PR TITLE
Update asset ownership as mandatory field

### LINKED ISSUE #128: /api/v1/user is no longer relevant since access is governed through the data partner account
Since we introduced the data partner concept we no longer have api tokens assigned to users. We now have data partner accounts which hold the tokens.

As such http://gresb.github.io/api-docs/#users is deprecated and needs to be removed.

### LINKED ISSUE #129: Asset ownership is now mandatory field for asset creation and api-doc (examples) should be updated
Sending an API request to create an asset fails in our Postman collection because asset ownership became a mandatory field and since our request body is missing it, validation error is raised. When asset has validation error on some mandatory fields it cannot be created. 422 unprocessed entity is returned as status code, and response does not include an assigned gresb_asset_id.

Therefore we need to update our docs and examples to reflect this change and send successful requests.

Screenshot 2023-02-16 at 16 55 22
Screenshot 2023-02-16 at 16 56 27

### LINKED ISSUE #130: Place certification & energy ratings specifications in a different section than patch asset data
http://gresb.github.io/api-docs/#asset-data-patch-entities-entity_id-assets-asset_id

It seems out of place that we specify how to create & update certifications in this section. Same for energy ratings. Better placed in a separate section which talks about certifications & energy ratings. For example http://gresb.github.io/api-docs/#asset-certifications.

image

### PR DESCRIPTION
**Pull request by GokceBaykan**
Also removes user endpoint and moves certification & energy ratings notices. Closes #128, Closes #129, Closes #130

### CODE REVIEW

**Review by joeyhorst — CHANGES_REQUESTED** [review]
We missed a section for clarifying `asset_ownership` as mandatory field. The user of notices seems slightly off to me. Lastly made some comments on existing text - I know I wrote most of it a while back, but looking at it with fresh eyes.

File: source/includes/_assets.md [hunk 1]
```diff
@@ -238,7 +238,9 @@ Returns the asset specified in the URL, along with its annual data (if available
 Creates a new asset for the specified entity in the URL. Returns the created
 asset, along with any validation errors and warnings.
 
-The `year` in _annual_data_ is required along with `asset_size`,`property_type_code` and `asset_name`.
+Some validation errors will prevent the asset from being created. Check the `gresb_asset_id` to verify that asset is created and unique ID is assigned.
+
+The `year` in _annual_data_ is required along with `asset_size`,`property_type_code`, `asset_name` and `asset_ownership`.
```
[the remark this answers is not in the document]
Comment (GokceBaykan) [reply source/includes/_assets.md:243]: I rechecked and you are right: [country, state_province, city] are mandatory fields. Changed the text to clarify on all the mandatory fields and if they need to be posted per annual data or year agnostically.

[the remark this answers is not in the document]
Comment (GokceBaykan) [reply source/includes/_assets.md:243]: It seems asset_ownership is not a mandatory field for always_create. [Asset](https://portal.dev.gresb.com/asset_portal/r/entities/21621-no-import/assets/901358/edit?validation_context=main) without this field will still be created.

Screenshot 2023-02-23 at 15 05 11

**Comment by GokceBaykan** [comment]
Updated the branch, it is ready for review again. @joeyhorst

**Review by joeyhorst — APPROVED** [review]
looks good 👍 thanks for making the style changes

**Pull request merged**